qgrad/qutip.py

```diff
@@ -0,0 +1,45 @@
+"""
+QuTiP functions that work with Jax
```

Comment:
We do not use any QuTiP functions imported from QuTiP. These are our own implementations. So it would be more appropriate to say "Standard quantum physics routines that work with JAX"